versions.tf

```diff
@@ -4,7 +4,13 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 6.0"
+      version = ">= 6.27"
     }
   }
+
+  provider_meta "aws" {
+    user_agent = [
+      "github.com/terraform-aws-modules"
```

Comment:
I'm realizing now our user-agent string parsing made some bad assumptions about the format of the "product name" section - namely that forward slashes wouldn't be used. 

I've got a fix up for this in https://github.com/hashicorp/aws-sdk-go-base/pull/1401, which I'm planning to get into the next set of provider releases (`v6.28.0` for the standard provider, slated for January 7, 2026). This should allow using the fully qualified repository name and version here.

```
github.com/terraform-aws-modules/terraform-aws-apigateway-v2/vX.Y.Z
```

Hopefully this enables a more granular view of usage by module in place of the generic organization name across all modules (if desired).